src/modules/endpoint/reducers/pushCategory.js

```diff
@@ -36,6 +37,7 @@ const pushCategory = (
 
   forEachTreeNode(root, node => categoriesMap.getChildren(node), depth, (node, children) => {
     resultModels[node.path] = node
+    Object.assign(resultModels, keyBy(children, child => child.path))
```

Comment:
I'm a bit unsure why we need to add the children. I mean they should be added by `resultModels[node.path] = node`

Reply:
I agree. I think all this line does is add 1 to the `depth`. This seems a bit odd to me.